troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js

```diff
@@ -16,11 +16,11 @@ define(function (require) {
 
     onDelete: function () {
       var project = this.props.project,
-        link = this.props.link;
+        external_link = this.props.link;
 
       modals.ExternalLinkModals.destroy({
-        link: this.props.link,
-        project: this.props.project
+        project,
+        external_link
```

Comment:
It give a searchable name to this, it is called "shorthand property names" and you'll find it mentioned as part of the [object initializer entry in Mozilla Developer Network docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer).